amgut/handlers/auth_handlers.py

```diff
@@ -38,6 +39,12 @@ def post(self):
         printresults = ag_data.checkPrintResults(skid)
         if printresults is None:
             printresults = 'n'
+
+        # make sure kit has bcrypt encrypted password, and fix if not
+        if not bcrypt.identify(kitinfo['password']):
```

Comment:
Under what circumstances does this function evaluate to `False`? The docstring on the function (`passlib.hash.bcrypt.identify`) is not helpful, and I couldn't find documentation [on their page](http://pythonhosted.org//passlib/lib/passlib.hash.bcrypt.html?highlight=bcrypt#passlib.hash.bcrypt).